update.sh

```diff
@@ -34,6 +34,9 @@ fi
 versions=( "${versions[@]%/}" )
 
 pipVersion="$(curl -fsSL 'https://pypi.org/pypi/pip/json' | jq -r .info.version)"
+getPipCommit="$(git ls-remote https://github.com/pypa/get-pip.git refs/heads/master | cut -d$'\t' -f1)"
```

Comment:
Can this be restricted to just `get-pip.py` so that the commit doesn't get updated on an unrelated change in one of the folders?

Reply:
Updated. :+1: